src/fmgc/src/flightphase/index.ts

```diff
@@ -0,0 +1,11 @@
+import { FlightPhaseManager } from './FlightPhaseManager';
+
+const flightPhaseManager = new FlightPhaseManager();
+
+export {
+    FlightPhaseManager,
+}
+
+export function getFlightPhaseManager(): FlightPhaseManager {
```

Comment:
This makes the flight phase manager a singleton, and available for use by typescript code (before it was only usable by the old js code).